### DISCUSSION TITLE
Wifi 6 USB adapter for R4

### DISCUSSION THREAD

**Discussion by chriseow** [WLAN-Pi/feedback#29] [Hardware - WLAN Pi R4] [answered]
From the link below, it says Wifi 6 is supported with USB adapters.
https://userguide.wlanpi.com/hardware/compare-wlan-pi-models

May I know which USB adapter is supported on the R4? Any plans to support Wifi 6E using a USB adapter?

**Comment by jiribrejcha** [comment, marked as answer]
Currently, CF-951AX is the best option for the R4 and it works out of the box. We tested Wi-Fi 6E on it too.

**Reply by chriseow** [reply]
ok thanks! will get that.

**Comment by adriel021** [comment]
hi anyone have used CF-957AX cant find CF-951AX here in the Philippines.

**Reply by joshschmelzle** [reply]
The CF-957AX is not supported due to being Realtek based.
